Dockerfile

```diff
@@ -13,17 +13,12 @@ RUN python3.9 -m pip install poetry \
  && python3.9 -m poetry install --without dev \
  && python3.9 -m poetry export -f requirements.txt --output requirements.txt --without-hashes
 
-
-# coverage test
-FROM poetry as coverage
-
+# run tests
 COPY example_plugin.py /app/
 COPY test_example_plugin.py /app/
 
-RUN mkdir /htmlcov \
- && python3.9 -m pip install coverage -r requirements.txt \
- && python3.9 -m coverage run test_example_plugin.py \
- && python3.9 -m coverage html -d /htmlcov --omit=/usr/local/*
+RUN pip3 install -r requirements.txt \
```

Comment:
Is that pip3 associated with python 3.6 or 3.9?

Reply:
I believe that should be `python3.9 -m pip install -r requirements.txt`.